packs/twitter/sensors/twitter_search_sensor.py

```diff
@@ -29,7 +29,7 @@ def setup(self):
 
     def poll(self):
         tso = TwitterSearchOrder()
-        tso.set_keywords([self._config['query']])
+        tso.set_keywords(self._config['query'])
```

Comment:
Since we don't have a schema for the config values, we should throw inside the constructor if the `query` value is not an array to prevent confusion for the users who are upgrading the pack.